src/entries/popup/pages/settings/customRPC/index.tsx

```diff
@@ -214,6 +249,29 @@ export function SettingsNetworksCustomRPC() {
               onBlur={onExplorerUrlBlur}
               borderColor={validations.explorerUrl ? 'accent' : 'red'}
             />
+            <Box padding="10px">
+              <Inline alignHorizontal="justify">
+                <Text
+                  align="center"
+                  weight="semibold"
+                  size="12pt"
+                  color="labelSecondary"
+                >
+                  {'Active'}
```

Comment:
i18n

Reply:
this is not final ui so i won't add it to languages